cmd/microservice-template/main.go

```diff
@@ -34,29 +34,26 @@ func main() {
 	defer func() { _ = tracerProvider.Shutdown(ctx) }()
 
 	// Creating handlers
-	productHandler := handlers.NewProductsHandler(logger)
+	queueHandler := handlers.NewQueueHandler(logger)
 
 	// Mux route handling with gorilla/mux
 	router := mux.NewRouter()
```

Comment:
See microservice-template to see how to refactor router creation to separate package